labgrid/remote/client.py

```diff
@@ -1516,6 +1521,8 @@ def main():
                            help="skip n 512-sized blocks at start of input")
     subparser.add_argument('--seek', type=int, default=0,
                            help="skip n 512-sized blocks at start of output")
+    subparser.add_argument('--mode', choices=WRITE_IMAGE_TOOLS.keys(), default='dd',
+                           help="Choose tool for writing images")
```

Comment:
With the change suggested above:
```suggestion
    subparser.add_argument('--mode', dest='write_mode',
                           type=USBStorageWriteMode, choices=USBStorageWriteMode, 
                           default=USBStorageWriteMode.DD,
                           help="Choose tool for writing images")
```